packages/react-components/src/components/table/constants.ts

```diff
@@ -0,0 +1 @@
+export const DEFAULT_PAGE_SIZE = 10;
```

Comment:
there is 20 in the above file and 10 here in this file,,, shouldnt they be same value ?

Reply:
default is set to 20 properties now.